charts/redpanda/values.schema.json

```diff
@@ -349,16 +349,142 @@
       "type": "object",
       "required": [
         "hostPath",
-        "persistentVolume"
+        "persistentVolume",
+        "tiered"
       ],
       "properties": {
         "hostPath": {
           "type": "string"
         },
+        "tiered": {
+          "type": "object",
+          "required": [
+            "mountType"
+          ],
+          "properties": {
+            "mountType": {
+              "type": "string",
+              "pattern": "^(none|hostPath|emptyDir|persistentVolume)$"
+            },
+            "hostPath": {
+              "type": "string"
+            },
+            "persistentVolume": {
+              "type": "object",
+              "required": [
+                "storageClass",
+                "labels",
+                "annotations"
+              ],
```

Comment:
None of this fields are required in persistent volume claim definition.

Reply:
The keys are required. This allows us to reference them as `.Values.storage.tiered.persistentVolume.storageClass` and not worry about dereferencing nulls instead of having to use dig.